Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -109,6 +110,18 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: _ViewController,
         
         return cell
     }
+    
+    public func collectionView(
+        _ collectionView: UICollectionView,
+        viewForSupplementaryElementOfKind kind: String,
+        at indexPath: IndexPath
+    ) -> UICollectionReusableView {
+        collectionView.dequeueReusableSupplementaryView(
+            ofKind: ListCollectionViewLayout.separatorKind,
+            withReuseIdentifier: separatorReuseIdentifier,
+            for: indexPath
+        ) as! _CellSeparatorReusableView<ExtraData>
```

Comment:
Why do you cast the type here? It should work without the cast, too, shouldn't it?